examples/brotli/docker-compose.yaml

```diff
@@ -1,15 +1,24 @@
 version: "3.7"
 services:
+
+  flask:
+    build:
+      context: ../shared/flask
+    image: flask:python-3.10-slim-bullseye
```

Comment:
How about envoy-flask, and comment it somewhere to let contributors use in examples.

Reply:
i think its important to _not_ include envoy in the name as it doesnt have envoy in the container and that would make it ambiguous

i was tempted to call it something shorter partly in terms of change - but in the end i just figured make it explicit

Reply:
>  and comment it somewhere to let contributors use in examples.


yep - good idea - there is a README for example contributors  - i can add to that

Reply:
> i think its important to _not_ include envoy in the name as it doesnt have envoy in the container and that would make it ambiguous
> 
> i was tempted to call it something shorter partly in terms of change - but in the end i just figured make it explicit

My concern is that do not mislead users that we use an official flask image, add it to doc may help.

Reply:
ok, i see - yep, hmmm

Reply:
maybe `flask_service:pyth...`

Reply:
The name seems longer,  since we do not often upgrade it, not many tags I think, maybe `flask:for-example`...

Reply:
i think the issue you raised above re `flask:` prefix was correct - that it implies coming from the flask namespace - so defo lets change to `flask_service` 

re the version - im minded to keep for now for reasons of being explicit - but its somewhat arbitrary what we call it so we can revisit in the future